apps/drec-api/src/pods/device-group/dto/device-group.dto.ts

```diff
@@ -104,4 +105,9 @@ export class DeviceGroupDTO implements IDeviceGroup {
   @IsArray()
   @IsOptional()
   labels: string[];
+
+  @ApiPropertyOptional({ type: [DeviceDTO] })
+  @IsArray()
+  @IsOptional()
+  devices?: IDevice[];
```

Comment:
Add `@ValidateNested()` decorator and it will also validate the individual `DeviceDTO`s in the array. 

Also, not sure why one has `DeviceDTO` type and the other has `IDevice`. I would make them both `DeviceDTO` to be consistent.